catalyst_ios/common.php

```diff
@@ -183,7 +183,16 @@ function scp_to_router($src, $dst)
 
 	sendexpectnobuffer(__FILE__.':'.__LINE__, $sms_sd_ctx, "conf t", "(config)#");
 	sendexpectnobuffer(__FILE__.':'.__LINE__, $sms_sd_ctx, "no ip scp server enable", "(config)#");
-	sendexpectnobuffer(__FILE__.':'.__LINE__, $sms_sd_ctx, "no username $login", "(config)#");
+
+	unset($tab);
+	$tab[0] = "(config)#";
+	$tab[1] = "[confirm]";
+	$index = sendexpect(__FILE__.':'.__LINE__, $sms_sd_ctx, "no username $login", $tab);
+	echo "$buffer \n";
```

Comment:
is this really needed?

Reply:
We reviewed the modification and found out that that line is unnecessary.
We will update the file and create another pull request.